src/mad_spark_alt/unified_cli.py

```
@@ -829,13 +829,14 @@ async def _run_qadi_analysis(
         if is_hybrid_mode:
             # is_hybrid_mode means documents/URLs detected, so using Gemini for all phases
-            print("🔀 Using Gemini for documents/URLs (auto mode)\n")
+            # Note: Hybrid orchestration (Gemini preprocess → Ollama QADI) not yet implemented
+            print("🔀 Using Gemini for documents/URLs (multimodal requires Gemini)\n")
         else:
             print(f"🤖 Using {provider_name} for analysis\n")
 
     except ValueError as e:
         print(f"❌ Error: {e}")
-        return
+        raise SystemExit(1)
```

Ruff (0.14.4)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)